ts/screens/profile/components/services/ServicesContactComponent.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { View } from "native-base";
+import { FlatList, ListRenderItemInfo } from "react-native";
+import { useState } from "react";
+import { GlobalState } from "../../../../store/reducers/types";
+import { Dispatch } from "../../../../store/actions/types";
+import { H1 } from "../../../../components/core/typography/H1";
+import { Body } from "../../../../components/core/typography/Body";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import { H4 } from "../../../../components/core/typography/H4";
+import IconFont from "../../../../components/ui/IconFont";
+import { IOColors } from "../../../../components/core/variables/IOColors";
+import ItemSeparatorComponent from "../../../../components/ItemSeparatorComponent";
+import { H5 } from "../../../../components/core/typography/H5";
+import TouchableDefaultOpacity from "../../../../components/TouchableDefaultOpacity";
+import I18n from "../../../../i18n";
+
+type Props = {
+  onSelectOption: (optionKey: string) => void;
+  hasAlreadyOnboarded?: true;
+} & ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+type ContactOption = {
+  title: string;
+  key: string;
+  description1: string;
+  description2?: string;
+};
+
+const options: ReadonlyArray<ContactOption> = [
+  {
+    title: I18n.t("services.optIn.preferences.quickConfig.title"),
+    key: "quick",
+    description1: I18n.t("services.optIn.preferences.quickConfig.body.text1"),
+    description2: I18n.t("services.optIn.preferences.quickConfig.body.text2")
```

Comment:
I suggest to use a more self-explanatory name: `description` and `details` makes me understand better where the fields are gonna be used without looking at the code below.

Reply:
They're indeed actually part of the same text, but needed 2 different texts in order to be rendered with different styles the first with Regular weight and the second Bold, instead.

Maybe a nested object could be more self explanatory?
```
description: {
  text1: string;
  text2?: string;
}
```

Reply:
Not sure, probably better to leave as it is then.